components/google-translate.tsx

```
@@ -10,30 +12,94 @@ declare global {
 }
 
 export function GoogleTranslate() {
+  const [isLoaded, setIsLoaded] = useState(false)
+  const [error, setError] = useState(false)
+
   useEffect(() => {
+    // Only run on client
+    if (typeof window === "undefined") return
+
+    const pollingRef = useRef<NodeJS.Timeout | null>(null)
```

Biome (2.4.4)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

(lint/correctness/useHookAtTopLevel)